specification/cognitiveservices/data-plane/MetricsAdvisor/preview/v1.0/MetricsAdvisor.json

```diff
@@ -6188,11 +6450,46 @@
       ],
       "properties": {
         "hookParameter": {
-          "$ref": "#/definitions/EmailHookParameter"
+          "$ref": "#/definitions/EmailHookParameterPatch"
         }
       },
       "x-ms-discriminator-value": "Email"
     },
+    "WebhookHookParameterPatch": {
+      "required": [
+        "endpoint"
+      ],
```

Comment:
Should we remove the required property in this patch definition?

Reply:
Yes, the WebhookHookParameterPatch and WebhookHookParameter were both wrong, the required should be applied on WebhookHookParameter. Fixed in the latest commit.